### ISSUE TITLE
[Bug] TimePicker widget don't show values without month indications

### ISSUE THREAD

**Issue by GilBr** [jawg/osm-contributor#57]
**Description:**
Values of opening_hours tag aren't shown if there isn't a month indication in the value.
For "Jan-Dec: Mon-Fr 12:00-16:00" => OK, for "Mon-Fr 12:00-16:00 => Not OK

**Steps and behaviour**
The problem appears on POI with opening_hours with existing values.
Look at this POI : [Coté Ardoise](https://www.openstreetmap.org/node/4370012347)

And it appears after POI edition with TimePicker widget without a month indication :
1. Select a POI (for ex. a restaurant)
2. Select the "Editer le POI" fonction
3. On opening_hours tag, use the TimePicker for days and hours
4. After tag edition, values are visible
5. Validate the modifications
6. Return to edition, no values are visible on the opening_hours tag

**Environment:**
Smartphone : Wiko Rainbow 4G – Android 4.4.2

**Version:**
3.0.0-beta3 + "Preset par défaut" profile + expert mode

Screenshots :
After step 3 : values are visible
![op_hours_ed1](https://cloud.githubusercontent.com/assets/21213907/18036032/30b3a154-6d62-11e6-9650-ebdd6235a64d.png)

After step 6 : no values are visible
![op_hours_ed2](https://cloud.githubusercontent.com/assets/21213907/18036038/4433a634-6d62-11e6-8133-d6b8a829977f.png)

**Comment by Hugoo13** [comment]
Thanks ! Effectively, this isn't a normal behavior. Also, there is a problem with icons (the blue square and the orange lozenge are not like that).

**Comment by GilBr** [comment]
I've tried the beta4 version but there are still problems with the widget. Hours are visible but shown in the month part. And the opening_hours tag is present a 2nd time on the old way with amenity tag. Is it normal ?

**Comment by loicortola** [comment]
This is probably an issue with the parser, will look into it today.

**Comment by GilBr** [comment]
It's better for me with the beta.5 version : time indications without month indications are shown, no more tags in double. And I have the good shape for blue and orange icons on the widget. 
But some time/date combinations aren't supported (and it's OK with the josm validator). The problem seems to come from comma separated values. See screenshots below : 
![screenshot_2016-09-20-20-37-39](https://cloud.githubusercontent.com/assets/21213907/18684016/fe8b0d52-7f72-11e6-9f2b-58ed617e21f2.png)
![screenshot_2016-09-20-20-38-03](https://cloud.githubusercontent.com/assets/21213907/18684026/0221f9e4-7f73-11e6-8dbb-977004b2fb92.png)

**Comment by GilBr** [comment]
And I'm not sure it's a good idea to use the Timepicker widget with the collection_times tag. 
The result is a bit strange ...
![screenshot_2016-09-20-20-57-01](https://cloud.githubusercontent.com/assets/21213907/18684999/93007758-7f76-11e6-9ec1-572fd7246a3a.png)

**Comment by Hugoo13** [comment]
You are right ! We are going to change the widget, with just days and hour. Months and "closed hour" are useless. 
I try to fix the "comma problem" too !

**Comment by Hugoo13** [comment]
Should be fixed :)
We are going to release the 3.0.0 final version. It should be available on the playstore in few hours.

**Comment by Hugoo13** [comment]
I close this issue. If you find a new problem or something you don't understand, please create a new issue. Thanks :)
